package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "glimpse.client.hud",
-  "version": "0.12.0",
+  "version": "0.14.1",
```

Comment:
Should this be 0.14.3 to match the others? I noticed they jumped a few versions at once.